app/lib/system/deploy.rb

```diff
@@ -5,14 +5,15 @@ module Deploy
     mattr_accessor :info
 
     class Info
-      attr_reader :revision, :deployed_at
-      attr_reader :release
+      VERSION = '2.x'
```

Comment:
I think this is better made a `private_constant` for the risk of it being used elsewhere. Usually project have a `VERSION` file somwhere for the purpose but it should be fine to be here for now I guess.